yarn-project/rollup-provider/src/app.ts

```diff
@@ -151,7 +151,7 @@ export function appFactory(node: AztecNode, prefix: string) {
     ctx.status = 200;
   });
 
-  router.get('/l1-l2-message-and-index', async (ctx: Koa.Context) => {
+  router.get('/l1-l2-message', async (ctx: Koa.Context) => {
```

Comment:
you have to change the URL in `http-node.ts` too here - https://github.com/AztecProtocol/aztec-packages/blob/master/yarn-project/aztec-node/src/aztec-node/http-node.ts#L250

Reply:
great catch